assets/models/system/codellama-34b-instruct-hf/spec.yaml

```diff
@@ -0,0 +1,31 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: codellama-34b-instruct-hf
+path: ./
+properties:
+  datasets: Publicly available sources
+  finetuning-tasks: text-classification
+  languages: EN
+  azureml.copyRegistryFilesToWorkspace: true
+tags:
+  Featured: ''
+  Preview: ''
+  inference_supported_envs:
+  - hf_tgi
+  evaluation_compute_allow_list:
+  - Standard_ND40rs_v2
+  - Standard_ND96asr_v4
+  - Standard_ND96amsr_v4
+  inference_compute_allow_list:
+  - Standard_ND96asr_v4
+  - Standard_ND96amsr_v4
+  finetune_compute_allow_list:
+  - Standard_ND96amsr_A100_v4
```

Comment:
Could we please confirm with @amangupta26 that FT can be enabled?

Reply:
Removed